### PR TITLE
Added print statement

### PR DESCRIPTION
**Pull request by AkanshuApphelix**

### CODE REVIEW

File: hello.py [hunk 1]
```diff
@@ -1,2 +1,2 @@
 print("hii world")
```
Comment (77ehtesham-alam) [inline hello.py:1]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

Comment (77ehtesham-alam) [inline hello.py:1]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

Comment (77ehtesham-alam) [inline hello.py:1]: File-level issue:
Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

Comment (77ehtesham-alam) [inline hello.py:1]: File-level issue:
Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

Comment (77ehtesham-alam) [inline hello.py:1]: Could not post inline comment for hello.py:1 (not part of changed lines).
Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Could not post inline comment for hello.py:1 (not part of changed lines).
Success: no issues found in 2 source files

Comment (77ehtesham-alam) [inline hello.py:1]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

Comment (77ehtesham-alam) [inline hello.py:1]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

Comment (77ehtesham-alam) [inline hello.py:1]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

File: hello.py [hunk 2]
```diff
@@ -1,2 +1,2 @@
...
-
```
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py [hunk 3]
```diff
@@ -1,5 +1,5 @@
 print("hii world")
-
```
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
+print("hii")
 x = 1
 y = 2
```
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py [hunk 5]
```diff
@@ -1,5 +1,5 @@
...
 z = 3
```
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: [main]	INFO	profile include tests: None
[main]	INFO	profile exclude tests: None
[main]	INFO	cli include tests: None
[main]	INFO	cli exclude tests: None

Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 1 above)
Comment (77ehtesham-alam) [inline hello.py:1]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 1 above)
Comment (77ehtesham-alam) [inline hello.py:1]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 1 above)
Comment (77ehtesham-alam) [inline hello.py:1]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 1 above)
Comment (77ehtesham-alam) [inline hello.py:1]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 1 above)
Comment (77ehtesham-alam) [inline hello.py:1]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 1 above)
Comment (77ehtesham-alam) [inline hello.py:1]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 1 above)
Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 1 above)
Comment (77ehtesham-alam) [inline hello.py:1]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:1]: Success: no issues found in 2 source files

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

Comment (77ehtesham-alam) [inline hello.py:2]: Success: no issues found in 2 source files

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 4 above)
Comment (77ehtesham-alam) [inline hi.py:4]: Constant name "x" doesn't conform to UPPER_CASE naming style

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style

File: hello.py (hunk 2 above)
Comment (77ehtesham-alam) [inline hello.py:2]: Missing module docstring

File: hi.py (hunk 3 above)
Comment (77ehtesham-alam) [inline hi.py:2]: Missing module docstring

File: hi.py (hunk 5 above)
Comment (77ehtesham-alam) [inline hi.py:5]: Constant name "y" doesn't conform to UPPER_CASE naming style
